test.js

```diff
@@ -13,18 +13,19 @@ const noop = () => {};
 
 const context = {
   library: {
-    name: '@expo/rudder-node-sdk',
+    name: 'expo-rudder-sdk-node',
     version,
   },
 };
 
 const metadata = { nodeVersion: process.versions.node };
+const host = 'http://localhost';
```

Comment:
extracted this so we can reference it in `flush - time out if configured`. could potentially spy on `client` for the info as well.